src/app/_components/home.tsx

```
@@ -2,99 +2,548 @@
+  const [isPanelOpen, setIsPanelOpen] = useIsPersonaPanelOpened();
+
+  const personaStore = usePersonaStore((state) => state);
+
+  const generate = async () => {
+    if (personaStore.isGenerationInProgress) return;
+    personaStore.setIsGenerationInProgress(true);
+
+    // @ts-ignore
+    const { personaId, ...response } = isSignedIn
+      ? await generatePersonaAction(prompt).catch((error) => {
+          if (error.message && error.message.includes("Insufficient tokens")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)